.gitattributes

```diff
@@ -0,0 +1,3 @@
+static/header_short.png filter=lfs diff=lfs merge=lfs -text
```

Comment:
`static/*.png` would be IMHO better (less verbose & less error prone).

Reply:
Actually I used `git lfs track --filename static/*.png` but it gave this result so I kept it.

Reply:
But I removed the file as LFS does not work for me in this repository. Is LFS allowed in repo @Indy2222 ?